staging/src/k8s.io/sample-controller/README.md

```diff
@@ -134,30 +134,17 @@ This feature is beta and enabled by default in v1.9.
 
 ### Example
 
-The schema in [`crd-validation.yaml`](./artifacts/examples/crd-validation.yaml) applies the following validation on the custom resource:
+The schema in [`crd.yaml`](./artifacts/examples/crd.yaml) applies the following validation on the custom resource:
 `spec.replicas` must be an integer and must have a minimum value of 1 and a maximum value of 10.
 
-In the above steps, use `crd-validation.yaml` to create the CRD:
-
-```sh
-# create a CustomResourceDefinition supporting validation
-kubectl create -f artifacts/examples/crd-validation.yaml
-```
-
 ## Subresources
 
-Custom Resources support `/status` and `/scale` subresources as a [beta feature](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#subresources) in v1.11 and is enabled by default.
-This feature is [alpha](https://v1-10.docs.kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#subresources) in v1.10 and to enable it you need to set the `CustomResourceSubresources` feature gate on the [kube-apiserver](https://kubernetes.io/docs/admin/kube-apiserver):
-
-```sh
---feature-gates=CustomResourceSubresources=true
-```
+Custom Resources support `/status` and `/scale` [subresources](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#subresources) from v1.16 in GA and are enabled by default.
```

Comment:
Both `CustomResourceValidation` and `CustomResourceSubresources` are GA. The feature gates have been removed in 1.18 so they cannot be disabled. Could you update the comment to remove the "and are enabled by default"?

Reply:
Oh sorry about that, will update the comment. Thanks!